### ISSUE TITLE
Including Symfony\Component\Form\Extension\Core\Type\ColorType in an admin form causes fatal JS error

### ISSUE THREAD

**Issue by molovo** [Kunstmaan/KunstmaanBundlesCMS#2501]
The following error appears in the console, which prevents any JS on the page from running.

```
jQuery.Deferred exception: n[e] is not a function e/<@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:13:31329
each@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:1:15288
each@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:1:13917
e@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:13:31159
value@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:24:14127
value@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:24:11720
value@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:23:27725
e@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:23:27132
s.default.fn[c]/i<@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:23:25537
each@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:1:15288
each@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:1:13917
s.default.fn[c]@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:23:25429
e@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:64:268
c/<@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:64:200
each@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:1:15288
each@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:1:13917
c@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:64:139
u@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:78:1363
i@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:78:651
@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:78:1706
f@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:2:9548
o/</p<@http://<redacted>/bundles/kunstmaanadmin/js/admin-bundle.min.js?v1:2:9851
```

Removing the field with ColorType causes the error to disappear.

Kunstmaan version: 5.3.1
Symfony version: 4.2.9
PHP version: 7.3.6

**Comment by mtnorthrop** [comment]
I also ran into this issue when I upgraded to Kunstmaan 5.3, so I've had to downgrade back down to 5.2 so that the color pickers in my admin section keep working. I wasn't able to identify the source of the problem while trying to debug the issue.

**Comment by dbeerten** [comment]
Fixed in 5.3.7
